### PR TITLE
Avatar Component Changes 

### PR DESCRIPTION
**Pull request by srsharon**

### CODE REVIEW

File: module/components/Avatar/Readme.md [hunk 1]
```diff
@@ -1,36 +1,14 @@
...
- 
- ```
-
-FontIcon Avatar:
-```js
-
-```
-
-FontIcon Avatar with custom colors and size:
+<Avatar color="info" src='styleguide/avatar_girl.jpg' />
```
Comment (jeradrutnam) [inline module/components/Avatar/Readme.md]: Please change `*_girl` to something neutral.

File: module/components/Avatar/Readme.md [hunk 2]
```diff
@@ -1,36 +1,14 @@
...
+  ```
 ```js
+<Avatar color="info" src='styleguide/avatar_boy.jpg' />
```
Comment (jeradrutnam) [inline module/components/Avatar/Readme.md]: Please change `*_boy` to something neutral.

File: module/components/Avatar/Readme.md [hunk 3]
```diff
@@ -1,36 +1,14 @@
...
+  ```
+  
+Two Letter Avatar:
 
-```
-
-SvgIcon Avatar:
 ```js
-
+<Avatar color="primary" labelText="Ad" />
```
Comment (jeradrutnam) [inline module/components/Avatar/Readme.md]: Add single label text example as well

Comment (jeradrutnam) [inline module/components/Avatar/Readme.md]: Add other color variations

File: module/components/Avatar/index.js [hunk 4]
```diff
@@ -18,47 +18,61 @@ const propTypes = {
...
+class Avatar extends Component {
     constructor(props) {
         super(props);
     }
 
-    render () {
-        let { backgroundColor,
+    render() {
+        let {
+            backgroundColor,
```
Comment (jeradrutnam) [inline module/components/Avatar/index.js]: Unused property

File: module/stylesheets/components/_avatar.scss [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  span {
+    color: #fff;
+    position: relative;
+    left: 14px;
+    width: 40px;
+    overflow: hidden;
+    text-align: center;
+    margin-left: -8px;
+  }
+  .avatar-rounded {
```
Comment (jeradrutnam) [inline module/stylesheets/components/_avatar.scss]: I believe this should be `&.avatar-rounded`

File: module/stylesheets/components/_avatar.scss [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    border-radius:50px;}
```
Comment (jeradrutnam) [inline module/stylesheets/components/_avatar.scss]: Please remove fixed pixel border radius. Use a fluid value like `50%`

File: module/stylesheets/components/_avatar.scss [hunk 7]
```diff
@@ -0,0 +1,23 @@
+.avatar {
+  font-size: 32px;
+  display: flex;
+  height: 50px;
+  width: 50px;
+
+  img {
+    height: 50px;
```
Comment (jeradrutnam) [inline module/stylesheets/components/_avatar.scss]: Move theses to variables. And try use "rem" instead "px"

**Review by jeradrutnam — CHANGES_REQUESTED** [review]
Please see inline comments

File: module/stylesheets/components/_avatar.scss (hunk 7 above)
Comment (srsharon) [reply module/stylesheets/components/_avatar.scss]: Will move this request to a new PR

File: module/stylesheets/components/_avatar.scss (hunk 5 above)
Comment (srsharon) [reply module/stylesheets/components/_avatar.scss]: Fixed

File: module/components/Avatar/Readme.md (hunk 1 above)
Comment (srsharon) [reply module/components/Avatar/Readme.md]: Fixed

File: module/components/Avatar/Readme.md (hunk 2 above)
Comment (srsharon) [reply module/components/Avatar/Readme.md]: Fixed

File: module/components/Avatar/Readme.md (hunk 3 above)
Comment (srsharon) [reply module/components/Avatar/Readme.md]: Fixed

Comment (srsharon) [reply module/components/Avatar/Readme.md]: Fixed

File: module/components/Avatar/index.js (hunk 4 above)
Comment (srsharon) [reply module/components/Avatar/index.js]: Fixed

File: module/stylesheets/components/_avatar.scss (hunk 6 above)
Comment (srsharon) [reply module/stylesheets/components/_avatar.scss]: Fixed

File: module/components/Avatar/index.js [hunk 8]
```diff
@@ -7,58 +7,52 @@ const propTypes = {
      * Useful to extend the style applied to components.
      */
     classes: PropTypes.string,
```
Comment (dakshika) [inline module/components/Avatar/index.js]: Shall we adopt other part. className: PropTypes.string,

File: module/components/Avatar/index.js [hunk 9]
```diff
@@ -7,58 +7,52 @@ const propTypes = {
...
-            size
+            labelText
         } = this.props;
 
-        let defaultClasses = 'avatar avatar-rounded';
+        let materialClasses = 'avatar avatar-rounded' + ((color) ? ' bg-' + color : '');
 
         return (
-            <div className={ classNames(classes, defaultClasses) } >{ icon }</div>
+            <div className={classNames(classes, materialClasses)}>
```
Comment (dakshika) [inline module/components/Avatar/index.js]: pass other attributes as well

**Review by dakshika — CHANGES_REQUESTED** [review]

File: module/components/Avatar/index.js [hunk 10]
```diff
@@ -6,59 +6,54 @@ const propTypes = {
...
     /**
-     * The src of the component. It's using the theme palette when that makes sense.
+     * If `true`, the button will use outline styling.
      */
-    src: PropTypes.string,
+    outline: PropTypes.bool,
+};
+
+const defaultProps = {
+    color: 'default',
```
Comment (dakshika) [inline module/components/Avatar/index.js]: isn't this primary?

**Review by dakshika — CHANGES_REQUESTED** [review]

File: module/components/Avatar/index.js (hunk 10 above)
Comment (srsharon) [reply module/components/Avatar/index.js]: Fixed, Added it to oneof

**Review by jeradrutnam — APPROVED** [review]
Approved with offline requested changes.

**Pull request merged**